build.sbt

```diff
@@ -394,17 +395,25 @@ lazy val scaldingParquet = module("parquet").settings(
       exclude("com.twitter.elephantbird", "elephant-bird-pig")
       exclude("com.twitter.elephantbird", "elephant-bird-core"),
     "org.apache.thrift" % "libthrift" % "0.7.0",
```

Comment:
Updated. The global thriftVersion is already present and set to 0.5.0, which I think should be okay.